asset/sass/styles.scss

```diff
@@ -0,0 +1,9 @@
+@import url(https://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700,700italic&subset=latin-ext,latin);
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,400,300,600,700&subset=latin-ext,latin);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville:400,400italic,700&subset=latin,latin-ext);
+
+@import 'base';
+@import 'zf-web';
+@import 'zf';
+@import '../node_modules/prismjs/themes/prism-okaidia.css';
```

Comment:
Not sure about this; I did a custom prism install with only the features we need for our site (I selected specific languages and extensions we needed).